napari/layers/image/image.py

```diff
@@ -742,7 +742,7 @@ def _raw_to_displayed(self, raw):
     def _is_async(self) -> bool:
         return True
 
-    def _make_slice_request(self, dims: Dims) -> _ImageSliceRequest:
+    def _make_slice_request(self, dims: Any) -> _ImageSliceRequest:
```

Comment:
this was a hack to make it work. need to revisit this, but I'm not sure how you want to go about it.

Reply:
Yeah, I think I ran into similar. As these are just type annotations, we can use forward references or possibly the `if TYPE_CHECKING` workaround.